docs/section23/Rule23-10.md

```diff
@@ -0,0 +1,53 @@
+
+# Airside System - Rule 23-10  
+
+**Rule ID:** 23-10  
+**Rule Description:** System 5-8 and 11 - part load VAV fan power shall be modeled using either method 1 or 2 in Table G3.1.3.15. This rule will only validate data points from Method-1 Part-load Fan Power Data.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**Appendix G Section Reference:** Section G3.1.3.15 VAV Fan Part-Load Performance (Systems 5 through 8 and 11)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8 or 11  
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** Table G3.1.3.15  
+**Function Call:**  
+
+1. baseline_hvac_type()  
+
+## Rule Logic:  
+
+- For each building segment in B_RMR: `for building_segment_b in B_RMR...building_segments:`
+
+  - Get HVAC system: `hvac_b = building_segment_b.heating_ventilation_air_conditioning_systems`
+
+    - Check if HVAC system is type 5, 6, 7, 8 or 11: `if baseline_hvac_type(hvac_b) in ["SYS-5", "SYS-6", "SYS-7", "SYS-8", "SYS-11"]:`
+
+      - Set applicability flag: `rule_applicability_check = TRUE`
+
+      - Get fan system serving HVAC system: `fan_system_b = hvac_b.fan_systems[0]` (See Note#1)
+
+        - For fan system supply, return and relief fan: `for fan_b in [fan_system_b.supply_fan, fan_system_b.return_fan, fan_system_b.relief_fan]:`
+
+          **Rule Assertion:**
+
+          - Case 1: For supply, return and relief fan in each fan system serving Baseline system type 5, 6, 7, 8 or 11, if fan part-load VAV is modeled as per Table G3.1.3.15: `if fan_b.output_validation_points[i] == data_lookup(table_G_3_1_3_15, i) for i in [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]: PASS`
+
```

Comment:
@xingcx we might need some guidance here on how to convert airflow to 0, 0.01, 0.02 etc data points.
Also need some guidance on what needs to be verified.
the RDS should look up the full load power (fan.design_electric_power) and multiply by the fraction in the table to show the expected wattage. Not sure how to best write it but a little more guidance on the required verification would be helpful.

Reply:
1. This RDS should apply to supply fan only.
2. It is my understanding that in the output_validation-points array we expect 11 sets of value pairs:
a. The first value in the pair would be the product of Fan Part-Load Ratio (first column in G3.1.3.15) by  Fan.design_airflow
b. The second value in the pair would be the product of Fraction of Full-Load Power (second column in G3.1.3.15) by Fan.design_electric_power,

Reply:
this looks very good now!

Reply:
I believe this the second value in each pair should be based of fan.design_electric_power, not fan.input_power. Input power is "Power delivered to the fan’s shaft and does not include the mechanical drive losses. Equivalent to fan brake
horsepower for inchpound units. Only used when specification_method is set to Detailed.